### PR TITLE
Adds two new types to produce relative timestamps and relative dates

### PR DESCRIPTION
**Pull request by wtekeu**
As requested by @binokaryg we introduce two new types: `rel-date` to input relative dates and `rel-timestamp` to produce relative timestamp. The offset is in number of days from the time when `medic-conf` is executed.

https://github.com/medic/medic-conf/issues/230

### CODE REVIEW

File: src/fn/csv-to-docs.js [hunk 1]
```diff
@@ -247,7 +247,9 @@ function parseColumn(rawCol, rawVal) {
     const type = parts[1];
     switch(type) {
       case 'date': val = new Date(rawVal); break;
+      case 'rel-date': val = new Date(new Date().getTime() + (parseInt(rawVal) * 24 * 60 * 60 * 1000)); break;
       case 'timestamp': val = parseTimestamp(rawVal); break;
+      case 'rel-timestamp': val = new Date().getTime() + (parseInt(rawVal) * 24 * 60 * 60 * 1000); break;
```
Comment (kennsippell) [inline src/fn/csv-to-docs.js]: should we put this calculation into a function?

**Comment by wtekeu** [comment]
according to @binokaryg
> Some dates definitely need to be in the future, such as appointment dates and expected date of delivery.

File: src/fn/csv-to-docs.js [hunk 2]
```diff
@@ -210,6 +210,10 @@ function parseBool(b) {
   else return b.toLowerCase() === 'true';
 }
 
+function calcRelTimestamp(b) {
```
Comment (kennsippell) [inline src/fn/csv-to-docs.js]: single letter variables (although still common in medic-conf) are not particularly desirable

**Comment by dianabarsan** [comment]
@ngaruko Should this PR be merged? 
I'm noticing that the associated issue (https://github.com/medic/medic-conf/issues/230) is closed.

**Pull request merged**